pkg/cmd/clients/factory.go

```diff
@@ -741,7 +741,7 @@ func (f *factory) CreateVaultOperatorClient() (vaultoperatorclient.Interface, er
 func (f *factory) CreateHelm(verbose bool,
 	helmBinary string,
 	noTiller bool,
-	helmTemplate bool) helm.Helmer {
+	helmTemplate bool) (helm.Helmer, error) {
```

Comment:
Would it be worth to split this change up and address first everything apart of this change to `CreateHelm`? We could explicitly ignore the error. This would reduce the changeset quite a bit.

One reason for that is that I believe that we really never (or at least rarely) jump into this branch which creates the error which we then propagate. Also imagining a future where we move towards Helm3 and all Tiller related code is obsolete, this particular piece of code might even go away. WDYT?

Reply:
That makes sense. You are right that once helm3 comes, this code would be removed. Fun fact: there are still 200+ places in the code base (excluding defer) where errors are not being checked, so I am going to focus on those now :)

Reply:
> Fun fact: there are still 200+ places in the code base (excluding defer) where errors are not being checked, so I am going to focus on those now 

Gee